Client/Frontend/Home/TopSites/DataManagement/FxHomeTopSitesManager.swift

```diff
@@ -95,8 +144,54 @@ class FxHomeTopSitesManager {
         }
         return pinnedSites
     }
+
+    // MARK: - Sponsored tiles (Contiles)
+
+    static let maximumNumberOfSponsoredTile = 2
+
+    // TODO: Check for settings user preference with https://mozilla-hub.atlassian.net/browse/FXIOS-3469
+    // TODO: Check for nimbus with https://mozilla-hub.atlassian.net/browse/FXIOS-3468
+    private var shouldLoadSponsoredTiles: Bool {
+        return featureFlags.isFeatureActiveForBuild(.sponsoredTiles) && profile.prefs.boolForKey(PrefsKeys.KeyShowSponsoredShortcuts) ?? true
```

Comment:
is `KeyShowSponsoredShortcuts` the user setting?

If so, we should toss it into the feature flag, because it already handles user settings. Lemme know if I can help set that up.

Or is that what that first TODO is referring to?

Reply:
In a way yeah, I added this `PrefsKeys.KeyShowSponsoredShortcuts` as a way to ignore sponsored shortcuts in UI tests (with LaunchArguments). It's not truly added as a settings pref and will need to be properly integrated with https://mozilla-hub.atlassian.net/browse/FXIOS-3469, hence why I left this TODO there